### PR TITLE
acc provision template side changes for qospolicy

### PR DESCRIPTION
**Pull request by niteshcha**

### CODE REVIEW

**Comment by niteshcha** [comment]
@tomflynn @snaiksat  please review this!

**Comment by jayaramsatya** [comment]
Hi Nitesh, this requires changes in all other flavors of  yaml files. 

We have a script for that to make changes automatically by taking your template file as input.

clone this repository https://github.com/noironetworks/support.git 

and run script support/acc-scripts/acc-provision/fix-testdata.sh on top of your changes.

**Comment by snaiksat** [comment]
@niteshcha could you kindly rebase your patch?

File: provision/Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ dist: test
 
 test:
 	flake8 --ignore E501,E731,E741,W504 acc_provision
-	python -m pytest acc_provision
+	python -m pytest -vv acc_provision
```
Comment (snaiksat) [inline provision/Makefile:14]: you probably used this for your debugging, can you please remove it?

Comment (niteshcha) [reply provision/Makefile]: done

**Comment by apoorva11029** [comment]
base_case_qos.inp.yaml looks the same as base_case.inp.yaml. If that's the case, we don't need to add another testcase here

**Comment by apoorva11029** [comment]
And do we need the qos template changes for all flavors?

**Comment by niteshcha** [comment]
> base_case_qos.inp.yaml looks the same as base_case.inp.yaml. If that's the case, we don't need to add another testcase here

done

**Comment by niteshcha** [comment]
> base_case_qos.inp.yaml looks the same as base_case.inp.yaml. If that's the case, we don't need to add another testcase here



> And do we need the qos template changes for all flavors?

yes

**Pull request merged**